src/Sylius/Bundle/ProductBundle/Form/EventSubscriber/ProductOptionFieldSubscriber.php

```diff
@@ -23,6 +24,19 @@
 class ProductOptionFieldSubscriber implements EventSubscriberInterface
 {
     /**
+     * @var VariantResolverInterface
+     */
+    private $defaultVariantResolver;
```

Comment:
It does not have to be a default one. Lets change `defaultVariantResolver` to `variantResolver` here, and in all other occurrences.